aces/joint_deconvolution/clean_HNCO_12m_7m.py

```diff
@@ -0,0 +1,91 @@
+            concat(vis       = glob.glob(workdir+'/'+sb_names['Region name'][i]+'/'+'*split.ms'),
+                   concatvis = workdir+'/'+sb_names['Region name'][i]+'/'+sb_names['Region name'][i]+'_12m_7m.concat.ms',
+                   freqtol   = '10MHz')
+
+    if os.path.exists(workdir+'/'+sb_names['Region name'][i]+'/'+sb_names['Region name'][i]+'_12m_7m.concat.ms') and not os.path.exists(workdir+'/'+sb_names['Region name'][i]+'/'+sb_names['Region name'][i]+'_12m_7m.HNCO.clean'):
+        hnco_pars = tclean_commands[sb_names['Region name'][i]+'_03_TM1']['tclean_cube_pars']['spw31']
+        tclean(vis                  = workdir+'/'+sb_names['Region name'][i]+'/'+sb_names['Region name'][i]+'_12m_7m.concat.ms',
+               field                = 'Sgr_A_star',
+               datacolumn           = 'corrected',
+               spw                  = '3, 9',
+               imagename            = workdir+'/'+sb_names['Region name'][i]+'/'+sb_names['Region name'][i]+'_12m_7m.HNCO.clean',
+               specmode             = 'cube',
+               gridder              = 'mosaic',
+               restoringbeam        = 'common',
+               calcpsf              = True,
+               calcres              = True,
+               restoration          = True,
+               pbcor                = True,
+               parallel             = True,
+               interactive          = 0,
+               cyclefactor          = 2.0,
+               mosweight            = hnco_pars['mosweight'],
+               outframe             = hnco_pars['outframe'],
+               intent               = hnco_pars['intent'],
+               imsize               = hnco_pars['imsize'],
+               cell                 = hnco_pars['cell'],
+               phasecenter          = hnco_pars['phasecenter'],
+               perchanweightdensity = hnco_pars['perchanweightdensity'],
+               width                = hnco_pars['width'],
+               start                = hnco_pars['start'],
+               nchan                = hnco_pars['nchan'],
+               deconvolver          = hnco_pars['deconvolver'],
+               weighting            = hnco_pars['weighting'],
+               robust               = hnco_pars['robust'],
+               niter                = hnco_pars['niter'],
+               threshold            = hnco_pars['threshold'],
+               usemask              = hnco_pars['usemask'],
+               sidelobethreshold    = hnco_pars['sidelobethreshold'],
+               noisethreshold       = hnco_pars['noisethreshold'],
+               lownoisethreshold    = hnco_pars['lownoisethreshold'],
+               negativethreshold    = hnco_pars['negativethreshold'],
+               minbeamfrac          = hnco_pars['minbeamfrac'],
+               growiterations       = hnco_pars['growiterations'],
+               dogrowprune          = hnco_pars['dogrowprune'])
```

Comment:
```suggestion
               **hnco_pars)
```

Reply:
@keflavich so does that just pass all of the parameters to tclean? And there will be no conflict will the previous manual overrides for e.g. `imagename`, `cyclefactor`, etc.?

Reply:
There are a few parameters that I need to specify manually: `imagename`, `calcpsf`, `calcres`, `spw`, etc, so there would be a conflict in those cases. I guess a workaround that would still result in more concise code would be simply to drop those entries from hnco_pars and pass the remaining, non-conflicting parameters ... ?

Reply:
yes, it doesn't seem like those parameters ought to be part of `hnco_pars` in that case - especially not `imagename`

Reply:
Okay. I have some updates to this script locally that I need to commit anyway, so I'll implement this and then commit.

Reply:
@keflavich I've implemented those changes and committed the updated version.